internal/provider/resource_vault_cluster.go

```diff
@@ -581,6 +581,8 @@ func setVaultClusterResourceData(d *schema.ResourceData, cluster *vaultmodels.Ha
 		} else {
 			d.Set("paths_filter", nil)
 		}
+	} else {
+		d.Set("paths_filter", nil)
```

Comment:
just for my knowledge: do we need a new test for this or by default as not set for non-replicated cluster ->`paths_filter` was nil already